kubernetes/yamls/development/api-server/api-deployment.yml

```diff
@@ -18,13 +18,8 @@ spec:
           httpGet:
             path: /api/status.json
             port: 80
-          initialDelaySeconds: 30
-          timeoutSeconds: 3
-        readinessProbe:
-          httpGet:
-            path: /api/status.json
-            port: 80
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
+          periodSeconds: 300
           timeoutSeconds: 3
```

Comment:
> `Liveness: http-get http://:80/api/status.json delay=90s timeout=3s period=300s #success=1 #failure=3`

This is not the failure message, it is just showing the configuration for probes.

You need to see the events for the pod in order to know if the probes are passing.